tests/RegressionTests.hs

```diff
@@ -309,6 +312,10 @@ nativeOpts :: Options
 nativeOpts = defaultOptions{useNativeArrays=True}
 nativeRetOpts :: Options
 nativeRetOpts = defaultOptions{useNativeReturns=True}
+woolOptions :: Options
+woolOptions = sicsOptions3
+openMPOptions :: Options
+openMPOptions = defaultOptions{platform = c99OpenMp}
```

Comment:
Do openMPOpts = c99OpenMpPlatformOptions so we know we aren't redefining the content of the options.

Reply:
Ok, missed that binding!